docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '3'
+
+services:
+
+  # PostgreSQL service
+  db:
+    image: postgres:latest
+    environment:
+      POSTGRES_DB: ${DJANGO_DB_NAME}
+      POSTGRES_USER: ${DJANGO_DB_USER}
+      POSTGRES_PASSWORD: ${DJANGO_DB_PASSWORD}
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+
+  # Django app service
+  core:
+    image: python:3.12-alpine
+    command: sh -c "pip install -r /app/requirements.txt && python /app/manage.py runserver 0.0.0.0:8000"
```

We are not running migrations here, i think you were right and we need a dockerfile already to add this command.

---

```suggestion
web:
    build: .
    ports:
      - "8000:8000"
    depends_on:
      - db
```
The docker-compose could stay like this and then add a dockerfile appart.